code/game/objects/structures/dresser.dm

```diff
@@ -5,6 +5,8 @@
 	icon_state = "dresser"
 	density = 1
 	anchored = 1
+	burn_state = FLAMMABLE
+	burntime = 25
```

Comment:
The problem with this is these aren't buildable, so once they're destroyed, that's it.

They likewise aren't wrenchable or destructable by default, so you can't just add it to the wooden planks build list, willy nilly either, or people will use it to block off areas, permanently.